tests/addresses/e2e/features/receive-money.feature

```diff
@@ -3,16 +3,42 @@ Feature: Receive money
 
   Background:
     Given I have completed the basic setup
-    And I have a "Test Wallet" rewards wallet with funds
-    And I have the following "Rewards" wallets:
+    And I have a "Test Wallet" wallet with funds
+    And I have the following wallets:
       | name          |
       | Target Wallet |
 
-  Scenario: Hide/show used addresses
+  @shelley
+  Scenario: Hide/show "Shelley" wallet used addresses
     Given I am on the "Target Wallet" wallet "receive" screen
     And I have made the following transactions:
       | source          | destination    | amount |
       | Test Wallet     | Target Wallet  | 1      |
     And I should see 1 used addresses
     When I click the ShowUsed switch
     Then I should not see any used addresses
+
+  @byron @api-wip-byron
+  Scenario: Hide/show "Byron" wallet used addresses
+    Given I am on the "Target Wallet" wallet "receive" screen
+    And I enter wallet password in generate address input field "Secret1234"
+    And I have 1 generated wallet addresses
+    And I have made the following transactions:
+      | source          | destination   | amount |
+      | Test Wallet     | Target Wallet  | 1     |
+    Then I should see 2 addresses
+    And I should see 1 used addresses
+    When I click the ShowUsed switch
+    Then I should see 1 addresses
+
+  @byron @api-wip-byron
```

Comment:
@tomothespian this test should be working as the "Receive" screen is fully functional on Byron builds.